_docs/management/space-egress.md

```diff
@@ -33,7 +33,8 @@ A summary of each of the ASGs that can be applied to your space are as follows:
 - ### `public-egress`
   - Requests being executed from within the space [can successfully be sent to the open internet]({{ site.baseurl }}{% link _docs/management/static-egress.md %}), our brokered services, and other internal routes you have created in your organization.
   - Applications can make requests to third party APIs.
-  - Applications can connect to our brokered services.
+  - Applications can connect to our brokered s3 service.
+  - Any requests to our other brokered services will be blocked.
```

Comment:
I'm not denying that this is true - but why did this change? Why can't apps talk to RDS via public egress? Is it because we want that traffic to go over the internal network and in fact, it has to because of the VPC?

Reply:
It didnt change.. I accidentally changed it and then reverted it here. All our RDS and Elastic-* are deployed in VPC to keep them isolated and private.

Reply:
Thats best practice in general for AWS, no services should be publicly accessible outside of your own APIs.